controllers/clusterclaims/clusterclaims-controller.go

```diff
@@ -135,7 +135,8 @@ func createManagedCluster(
 		if _, ok := newLabels["name"]; !ok {
 			newLabels["name"] = claimName
 		}
-		newLabels["vendor"] = "OpenShift" // This is always true
+		newLabels["vendor"] = "OpenShift"  // This is always true
+		newLabels["cloud"] = "auto-detect" //This is used to detect cloud provider, like: GCP,AWS
```

Comment:
Just an observation, I was seeing this label get propagated without auto-detect being present.